<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>note_creator</class>
 <widget class="QMainWindow" name="note_creator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>420</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Note Creator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="launch_button">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>340</y>
      <width>356</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Launch Note</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="note_name_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>291</y>
      <width>361</width>
      <height>39</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>Type filename and extension here.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="editor_selection_dropdown_box">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>341</y>
      <width>261</width>
      <height>39</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>notepad</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Text editor:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="favorite_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>New Path 0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Path 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Path 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Path 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Path 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Path 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Path 6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Path 7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Path 8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Path 9</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="directory_line_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Directory selected with &quot;Open Directory&quot; will display here. Highlight one of the lines above then press &quot;Set Directory&quot; to save selected directory.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_button">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>290</y>
      <width>176</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Set Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>290</y>
      <width>176</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Open Directory</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNote_Creator">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="dropdown_file_exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Note_Creator"/>
   </widget>
   <addaction name="menuNote_Creator"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="dropdown_file_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Note_Creator">
   <property name="text">
    <string>About Note Creator</string>
   </property>
  </action>
  <action name="actionSave_and_Close">
   <property name="text">
    <string>Save and Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
